packages/smooth_app/lib/pages/product/nutrition_page_loaded.dart

```diff
@@ -265,18 +243,24 @@ class _NutritionPageLoadedState extends State<NutritionPageLoaded>
             ),
           ),
         );
+
+        if (_nutrientToHighlight == orderedNutrient) {
+          onNextFrame(() => _focusNodes[i].requestFocus());
```

Comment:
I've already seen strange situations where `i`, being not final, changed values before the method being called, so I suggest you use `final int index = i;` and `_focusNodes[index]` instead.

Reply:
Nice catch